reference-service/src/main/java/com/transformuk/hee/tis/reference/service/api/RoleResource.java

```diff
@@ -244,6 +244,46 @@ public RoleResource(RoleRepository roleRepository, RoleMapper roleMapper,
     return new ResponseEntity<>(result, HttpStatus.OK);
   }
 
+  /**
+   * EXISTS /roles/matches/ : check if there's a role in the database that matches the code
```

Comment:
I'd say 'EXISTS' should be replaced with 'POST' here. I've not seen the term 'EXISTS' before in a javadoc - it sort of makes sense to me in the rolesExists() definition, but all the other javadocs here simply include the HTTP method type, which might be better to stick with in this instance.

Reply:
True